src/widgets/ot_sticky.eliomi

```diff
@@ -16,6 +16,27 @@ val supports_position_sticky : 'a elt -> bool
     due to position:sticky polyfill [make_sticky] *)
 val is_sticky : 'a elt -> bool
 
+(** determines how certain values of [make_sticky] are computed
+    [`Fix] sets the value to a inline length when the element get stuck.
+    With [`Leave] the value is not touched at all.
+    [`Sync] causes the value to continuously (on window scroll/resize) be
+    derived from the inline other element, meaning from the inline for
+    the element fixed, and from fixed for the inline.
+*)
```

Comment:
I don't understand this comment. Yesterday I thought that it was remnant from an old version, so I removed it. What is it that _"determines how..."_? Where do values like `` `Fix`` and `` `Sync`` appear?